FeedStoreChallenge/CachedFeedStoreImage.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CachedFeedStoreImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Hoff Henry Pereira da Silva on 27/07/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(CachedFeedStoreImage)
+final class CachedFeedStoreImage: NSManagedObject {
+	@NSManaged internal var id: UUID
+	@NSManaged internal var imageDescription: String?
+	@NSManaged internal var location: String?
+	@NSManaged internal var url: URL
```

Comment:
`internal` is a default access level, no need to define it explicitly. Please remove `internal` modifiers

```suggestion
	@NSManaged var id: UUID
	@NSManaged var imageDescription: String?
	@NSManaged var location: String?
	@NSManaged var url: URL
```